### ISSUE TITLE
Running on a networked host? 

### ISSUE THREAD

**Issue by daveseff** [pingidentity/pingidentity-cdr-sandbox#67]
From the doco, it seems that running the sandbox assumes everything is on the local host. The docker containers only listen on the lo interface, which doesn't do much for us as we are trying to run on a networked linux host. Is there an option for that?

**Comment by pdang-ping** [comment]
Hi David,

By design all containers are not accessible externally.   You would need to expose the ports (which has already been done in the docker-compose.yml file).

Please run docker-compose ps and you will be able to ascertain what ports are opened.

Please see the documentation here:
https://cdr.pingidentity.com/ for further details and how to change the cdr.env file to meet your requirements.

Cheers,

**Comment by daveseff** [comment]
Thanks for the RTFM response.  
If I update BASE_HOSTNAME and BASE_URL to the FQDN on my host, Will that make the application accessible on my network?

**Comment by pdang-ping** [comment]
Hi David - the doco is very clear about this.

The BASE_HOSTNAME value represents the base DNS name used to access the Data Holder's web services and applications. You will need to ensure hosts files or DNS are updated to reflect the appropriate IP addresses for the containers once the sandbox is running if you change this value.

As per CDR requirements some endpoints require MTLS - as such hostnames have to match. 

Again please RTFM - as there will be other values you need to update.  Or have a look at the cdr.env file and you will get a feel of what values need to be update - search for ones that have .local at the end of the URL.

**Comment by ttranatping** [comment]
Our customers and us have been successful deploying on linux on a remote service, however our setup may be different to yours. There may be hardening and firewall variances in your environment to what we have worked with.

It may be worth noting that I have come across an issue on some linux platforms in relation to ipv6. This was resolved by manually installing Docker to a newer version. The Docker version installed using the distribution's package installer installed a very old version of Docker. If you are using a package installer to install Docker, I suggest you find alternative instructions for your distribution.

Being a community service, our support for the CDR Sandbox is limited and the sole intent is to demonstrate CDR functioning to the public. This objective can be achieved on localhost. We will have to close this ticket as it is a Docker and Linux related query.
